html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Forms</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <p>
+  <div>
+    <label for="email">Email: </label>
+    <div>
+      <input type="text" id="email" name="email">
+    </div>
+  </div>
+  <div>
+    <label for="coding">Why is coding cool? </label>
+    <div>
+      <select name="coding">
+        <option value="select-an-option">Select an Option</option>
+        <option value="beacuse-its-fun">Because it's fun</option>
+        <option value="because-its-cool">Because it's cool</option>
+        <option value="because-its-powerful">Because it's Powerful</option>
+      </select>
+    </div>
+  </div>
+  <div>
+    <div>
+      <label for="language">Favorite Language (choose one)</label>
+      <div>
+        <input type="radio" name="language" value="html" />HTML
```

Comment:
You should put the label around the text for each option. For instance, "HTML" is the text that should be the label for this input 👍